tests/wpunit/Tribe/Events/Models/Event_ModelTest.php

```diff
@@ -46,6 +48,28 @@ public function it_should_return_past() {
 		$this->assertTrue( $event->is_past );
 	}
 
+	/**
+	 * @test
+	 */
+	public function it_should_return_is_now() {
+		$mock  = $this->get_mock_event( 'events/single/1.json' );
+		$event = Event::from_post( $mock )->to_post();
+
+		$this->assertFalse( $event->is_now );
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_return_is_now_true() {
+		// Here, we're just overriding the function in common to assure it returns true.
```

Comment:
I would not do this: while there is some value in testing that, given the `is_now` from common will return a value the `$event->is_now` property will reflect that, there is more value in writing a real integration test where an Event is set up to happen now, using `tribe_events()->set_args( ... )->create()` and test what comes out of it.